scripts/run_all_benchmarks.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -e
+
+# Runs all benchmarks for all pallets, for the Substrate node.
+# Should be run on a reference machine to gain accurate benchmarks
+# current reference machine: https://github.com/paritytech/substrate/pull/5848
+
+echo "[+] Running all benchmarks for Substrate"
+
+if [ $1 != "skip-build" ]
+then
+  cargo +nightly build --profile production --locked --features=runtime-benchmarks
+fi
+
+./target/production/substrate benchmark pallet \
+    --chain=dev \
+    --list |\
+  tail -n+2 |\
+  cut -d',' -f1 |\
+  uniq > "substrate_pallets"
```

Comment:
Using a file here to store a temporary variable :see_no_evil:   
Let me figure out if this is possible w/o.

Reply:
danke